plugins/woocommerce-blocks/assets/js/base/components/product-price/style.scss

```diff
@@ -23,6 +23,10 @@
 	ins {
 		text-decoration: none;
 	}
+
+	.woocommerce-Price-amount {
```

Comment:
Is the uppercase `Price` intended here? 🤔

Reply:
Hmmm, yes, that's a class we inherited from old templates, and it has been with the uppercase `P` from before blocks excited. We kept it the same to avoid breaking existing themes.